classes/ObjectModel.php

```diff
@@ -2155,7 +2155,7 @@ public function setFieldsToUpdate(?array $fields)
     }
 
     /**
-     * @return array|null
+     * @return array<string, bool|array<int, bool>>|null
      */
     public function getFieldsToUpdate(): ?array
```

Comment:
Is it still used?
You might have added it in early stages (when I added it in another PR and you might have duplicated this to avoid blocking the PR). but I think we don't use it anymore

Reply:
I think you meant the former `addFieldsToUpdate` which has been removed?

Reply:
oh you might be right, sorry

Reply:
No it's ok, I just think this is an old comment that stayed even though the code has been removed (it was indeed present in previous versions of the PR)